synapse/events/spamcheck.py

```diff
@@ -259,6 +265,24 @@ async def check_event_for_spam(
 
         return False
 
+    async def user_may_join_room(self, userid: str, room_id: str, is_invited: bool):
```

Comment:
It makes me somewhat sad that we seem to use `userid` in this file whereas eveywhere else in synapse we use `user_id`

Reply:
Yeah fair. This was just copy-pasta that I forgot to fix.

```suggestion
    async def user_may_join_room(self, user_id: str, room_id: str, is_invited: bool):
```